test/fixtures/node_revisions.yml

```diff
@@ -114,7 +128,7 @@ hashtag_in_header:
   uid: 1
   title: grabs hashtags in headers
   body: |+
-    # Heading 1 is about #hashtags 
+    # Heading 1 is about #hashtags
```

Comment:
I believe this change may be triggering the bug. Tough one, and really it shouldn't be -- but the regex driving the hashtag splicing is not as broad as we'd like: that's here: https://github.com/publiclab/plots2/blob/master/config/initializers/constants.rb#L3

If we try to fix this, I'd strongly prefer opening a new issue, and re-introducing this error in the test in a separate PR, so as to respect the one-issue-per-PR convention. But I do think it's one we'd really love to fix! 

Thanks!

Reply:
Oh wow, okay. I think that was my text editor auto-squashing anything it considers whitespace on file save. Let me add that trailing space back in, re-push, and see what happens.

If you'd like, I'd be happy to tackle the regex change as well.